ansible/dimensiondata/dimensiondata_backup_client.py

```diff
@@ -14,7 +14,7 @@
 
 DOCUMENTATION = '''
 ---
-module: dimensiondata_backup
+module: dimensiondata_backup client
```

Comment:
This matches the name of the actual module (backup vs backup client).
Shouldn't it match the filename